src/fiatExchanges/ReviewScreen.tsx

```diff
@@ -0,0 +1,180 @@
+import { RouteProp } from '@react-navigation/native'
+import { StackScreenProps } from '@react-navigation/stack'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, SafeAreaView, StyleSheet, Text, View } from 'react-native'
+import BackButton from 'src/components/BackButton'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import LineItemRow from 'src/components/LineItemRow'
+import i18n from 'src/i18n'
+import { emptyHeader } from 'src/navigator/Headers'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import variables from 'src/styles/variables'
+import { CICOFlow } from './utils'
+
+type Props = StackScreenProps<StackParamList, Screens.FiatConnectReview>
+
+export default function FiatConnectReviewScreen({ route, navigation }: Props) {
+  const { t } = useTranslation()
+
+  // TODO(satish): get all hardcoded fields from props
+  const { flow } = route.params
+
+  return (
+    <SafeAreaView style={styles.content}>
+      <View>
+        <View style={styles.amountContainer}>
+          <Text style={styles.amountText}>20.01 cUSD</Text>
+        </View>
+        <TransactionDetails flow={flow} />
+        <PaymentMethod />
+      </View>
+      <Button
+        style={styles.submitBtn}
+        type={BtnTypes.PRIMARY}
+        size={BtnSizes.FULL}
+        text={
+          flow === CICOFlow.CashIn
+            ? t('fiatExchangeFlow.cashIn.fiatConnectReviewButton')
+            : t('fiatExchangeFlow.cashOut.fiatConnectReviewButton')
+        }
+        onPress={() => {}}
+      />
+    </SafeAreaView>
+  )
+}
+
+function TransactionDetails({ flow }: { flow: CICOFlow }) {
+  const { t } = useTranslation()
+  return (
+    <View style={styles.sectionContainer}>
+      <Text style={styles.sectionHeaderText}>
+        {t('fiatConnectReviewScreen.transactionDetails')}
+      </Text>
+      <LineItemRow
+        style={styles.sectionMainTextContainer}
+        textStyle={styles.sectionMainText}
+        title={
+          flow === CICOFlow.CashIn
+            ? t('fiatExchangeFlow.cashIn.transactionDetailsAmount')
+            : t('fiatExchangeFlow.cashOut.transactionDetailsAmount')
+        }
+        amount="$20.00"
+      />
+      <LineItemRow
+        title="25 cUSD @ $1.00"
+        amount="20.0010 cUSD"
+        style={styles.sectionSubTextContainer}
+        textStyle={styles.sectionSubText}
+      />
```

Comment:
Anytime you display something in Fiat Currency you should use the CurrencyDisplay component, and for crypto currency the TokenDisplay Component.

Take a look at the [FiatExchangeAmount component](https://github.com/valora-inc/wallet/blob/main/src/fiatExchanges/FiatExchangeAmount.tsx#L274-L290) it has a UI that looks very similar.

Reply:
What that component looks like
![IMG_2358](https://user-images.githubusercontent.com/8432644/169620335-922c77fc-76ff-4af8-a843-e9bf303f0282.PNG)